PCbuild/readme.txt

```diff
@@ -202,15 +209,18 @@ interpreter, but they do implement several major features.  See the
 "Getting External Sources" section below for additional information
 about getting the source for building these libraries.  The sub-projects
 are:
+
 _bz2
     Python wrapper for version 1.0.8 of the libbzip2 compression library
     Homepage:
         http://www.bzip.org/
-_lzma
+
+liblzma
```

Comment:
I think changing these from `_lzma`, `_ssl`, and `_sqlite3` to `liblzma`, `openssl`, and `sqlite3` is correct, as the former projects are the Python modules and the latter are the wrappers for external libraries, which is what this section describes.

Reply:
They're still described as "Python wrapper for ...", so should still be the names of the vcxproj file in PCbuild?

Reply:
Would it make sense to list both? Eg "openssl, _ssl"

Reply:
It's worth pointing out the additional builds, and makes sense to bundle them together. So yeah, but put the wrapper first and then mention the library project in the text.